### DISCUSSION TITLE
How to push data change from server to client?

### DISCUSSION THREAD

**Discussion by lukaszimmermann1** [FreeOpcUa/python-opcua#1336] [Q&A]
Hello,

I'm new to OPC UA and am working on a system where a Client ('Sensor') continuously streams data to a Server. Another Client ('Detector') is then supposed to work with the streamed data.

Right now, I have set up a subscription handler within the Detector which subscribes to the node value corresponding to the streamed data and retrieves it every x milliseconds.
What I want to implement is a solution where the Detector does not have to continuously ask the server whether there have been data changes but instead make it so that the Server notifies Detector whenenever a change has occured. Is this possible within the python-opcua framework?

Thanks and have a nice day!

**Comment by AndreasHeine** [comment]
https://reference.opcfoundation.org/v104/Core/docs/Part4/5.13.1/

**Reply by oroulet** [reply]
and look at examples for how to use subscriptions
